### DISCUSSION TITLE
Higher CPU usage in v 1.18.x

### DISCUSSION THREAD

**Discussion by Gr4ffy** [Koenkk/zigbee2mqtt#6747] [General]
Hi,

I noticed higher CPU usage in v 1.18.x (Xeon E3-1260L @ 2.40GHz)
Z2M,HA & Mosquitto in docker containers. Coordinator - Conbee II - firmware 0x26680700 30-Nov-2020
node index.js
1.17.x in htop shows ~1%
1.18.x ~5%

https://streamable.com/6i6uld
https://streamable.com/kagxxj

**Comment by canton7** [comment]
I noticed something similar. It's now taking ~10% of the CPU time of a Raspberry Pi 4, continuously. Running on bare-metal linux.

**Comment by roumano** [comment]
Maybe i found the same kind of issue ? :
For me, it's related to wakeup via this kind of (found via strace ) `clock_gettime(CLOCK_MONOTONIC, {tv_sec=125522, tv_nsec=772175}) = 0`
more info here : https://github.com/Koenkk/zigbee2mqtt/discussions/6946

**Comment by ChrisHae** [comment]
pls see https://github.com/Koenkk/zigbee2mqtt/issues/7004#issuecomment-819501729

**Comment by Gr4ffy** [comment]
Thanks @ChrisHae 
Looks like solved [here](https://github.com/Koenkk/zigbee2mqtt/issues/7004#issuecomment-819621727)
